Classes/gitx_askpasswd_main.m

```diff
@@ -171,7 +171,10 @@ int	main( int argc, const char** argv )
 		
 		[defaults synchronize];
 		
-		return code;
+		if (code <= INT_MAX) {
+			return (int)code;
+		};
+		return INT_MAX;
```

Comment:
I put in a constant for the codes returned by the event loop (there are currently custom codes, it seems) and explicitly return 0 or 1 depending on whether the dialog returns OK or Cancel.